pkg/runtime/kubernetes/runtime_getter.go

```diff
@@ -36,24 +37,24 @@ import (
 )
 
 func (k *KubeadmRuntime) getKubeVersion() string {
-	return k.KubeadmConfig.ClusterConfiguration.KubernetesVersion
+	return k.kubeadmConfig.ClusterConfiguration.KubernetesVersion
 }
 
 // old implementation doesn't consider multiple rootfs images; here get the first rootfs image
 func (k *KubeadmRuntime) getKubeVersionFromImage() string {
-	img := k.Cluster.GetRootfsImage()
+	img := k.cluster.GetRootfsImage()
 	if img == nil || img.Labels == nil {
 		return ""
 	}
 	return img.Labels[v1beta1.ImageKubeVersionKey]
 }
 
 func (k *KubeadmRuntime) getMaster0IP() string {
```

Comment:
What is the meaning of writing it as a function? I think it can be called directly, unless some functions that require logic processing can be encapsulated